Last-Update: 2013-04-03
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: Use default editor instead of vi

--- a/misc/mc.ext.in
+++ b/misc/mc.ext.in
@@ -321,9 +321,9 @@
 regex/i/\.(s|asm)$
 	Include=editor
 
 include/editor
-	Open=%var{EDITOR:vi} %f
+	Open=%var{EDITOR:editor} %f
 
 # .so libraries
 regex/\.(so|so\.[0-9\.]*)$
 	View=%view{ascii} @EXTHELPERSDIR@/misc.sh view so
